src/test/java/org/mskcc/cmo/metadb/ValidRequestCheckerTest.java

```diff
@@ -0,0 +1,112 @@
+    }
+    
+    /**
+     * Test for handling of fields that are a mix of null or empty strings in the requestJson
+     * The filter is expected to fail
+     */
+    @Test
+    public void testValidRequestJson() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest1JsonDataWith2T2N");
+        String modifiedRequestJson = requestJson.getJsonString();
+        Assert.assertNotNull(modifiedRequestJson);
+    }
+
+    /**
+     * Test for handling all null or empty fields in requestJson
+     */
+    @Test
+    public void testNullJsonFieldHandlingInRequestJson() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest4JsonNullOrEmptyValues");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNull(modifiedRequestJson);
+    }
+    
+    /**
+     * Test for handling request with no valid samples
+     */
+    @Test
+    public void testRequestJsonWithNoValidSamples() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest1cJsonDataWithMAS");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNull(modifiedRequestJson);
+    }
+    
+    /**
+     * Test for handling request with 2 valid samples and 2 invalid samples
+     * 
+     */
+    @Test
+    public void testRequestJsonWithTwoInvalidSamples() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest1dJsonDataWithMFS");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNotSame(modifiedRequestJson, requestJson.getJsonString());
+    }
+    
+    /**
+     * Test for handling request with available fields to determine sample type abbreviation
+     * 
+     */
+    @Test
+    public void testRequestJsonWithMissingFieldsUsedForSTA() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest1aJsonDataWithSTA");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNotNull(modifiedRequestJson, requestJson.getJsonString());
+    }
+    
+//    /**
+//     * Test for handling request with available fields to determine nucleic acid abbreviation
+//     * 
+//     */
+//    @Test
+//    public void testRequestJsonWithMissingFieldsUsedForNAA() throws Exception {
+//        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockIncomingRequest1aJsonDataWithNAA");
+//        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+//        Assert.assertNotNull(modifiedRequestJson, requestJson.getJsonString());
+//    }
```

Comment:
does this still need to be here or is it just commented out temporarily? 

If temporary then you should add a TODO or some note stating it's intentional and what the action item is to put this in place